### PR TITLE
Fix ad response encoding

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @JordanMilne 

on https://m.staging.wireddit.com/

### CODE REVIEW

File: src/views/components/Ad.jsx [hunk 1]
```diff
@@ -77,7 +78,12 @@ class Ad extends BaseComponent {
     }
 
     return new Promise((resolve, reject) => {
-      superagent.post(origin + this.props.adsPath)
+      var adUrl = url.format({
+        host: origin,
+        path: this.props.adsPath,
+        query: { raw_json: true },
```
Comment (JordanMilne) [inline src/views/components/Ad.jsx]: nit: raw_json can be either in the postData or the query params. It'd be more idiomatic to put it in the postData here

Comment (JordanMilne) [inline src/views/components/Ad.jsx]: Also, `raw_json` must be `=== "1"`

**Comment by JordanMilne** [comment]
:nail_care:

**Comment by dwick** [comment]
:haircut:

**Comment by JordanMilne** [comment]
:fish:

**Pull request merged**